gemrb/core/GameData.h

```diff
@@ -136,7 +136,7 @@ class GEM_EXPORT GameData : public ResourceManager
 	Cache ItemCache;
 	Cache SpellCache;
 	Cache EffectCache;
-	std::unordered_map<const ResRef, PaletteHolder, ResRef::Hash> PaletteCache;
+	std::unordered_map< ResRef, PaletteHolder, ResRef::Hash> PaletteCache;
```

Comment:
```suggestion
	std::unordered_map<ResRef, PaletteHolder, ResRef::Hash> PaletteCache;
```

Reply:
Sorry, sloppy habit, I fixed and updated